main/src/cgeo/geocaching/maps/CGeoMap.java

```diff
@@ -525,7 +528,20 @@ private void prepareFilterBar() {
     @Override
     public void onResume() {
         super.onResume();
-        resumeDisposables.addAll(geoDirUpdate.start(GeoDirHandler.UPDATE_GEODIR), startTimer());
+
+        // resume location access
+        PermissionHandler.executeIfLocationPermissionGranted(this.activity, new PermissionGrantedCallback(PermissionRequestContext.CacheDetailActivity) {
+
+            @Override
+            public void execute() {
+                Log.d("CacheDetailActivity.OnResumePermissionGrantedCallback.execute");
```

Comment:
copy paste? CacheDetailActivity?

Reply:
Thanks for catching. Yes, copy paste from CacheDetailActivity. I was thinking how to encapsulate that, as most of the code is the same (only line 542 is specific for this location.